README.md

```diff
@@ -152,7 +152,7 @@ There is a dummy user named "Server" in every player's friends list that you can
  
 # Quick Troubleshooting
 
-* If compiling wasn't successful, please check your JDK installation (JDK 17 and validated JDK's bin PATH variable)
+* If compiling wasn't successful, please check your JDK installation (JDK 17 and validated JDK's bin PATH variable.)

```

Comment:
why the . is
inside those brackets

Reply:
this is a joke pr lmao

Reply:
add some . to get contributor 100% works

Reply:
> add some . to get contributor 100% works

can actually be a contributor if the dot was actually in its right place

Reply:
. moment

Reply:
def not resolved